gensim/sklearn_integration/sklearn_wrapper_gensim_ldaseqmodel.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Scikit learn interface for gensim for easy use of gensim with scikit-learn
+Follows scikit-learn API conventions
+"""
+
+from sklearn.base import TransformerMixin, BaseEstimator
+
+from gensim import models
+from gensim.sklearn_integration import base_sklearn_wrapper
+
+
+class SklearnWrapperLDASeqModel(base_sklearn_wrapper.BaseSklearnWrapper, TransformerMixin, BaseEstimator):
+    """
+    Base LdaSeq module
+    """
+
+    def __init__(self, time_slice=None, id2word=None, alphas=0.01, num_topics=10,
+                initialize='gensim', sstats=None, lda_model=None, obs_variance=0.5, chain_variance=0.005, passes=10,
+                random_state=None, lda_inference_max_iter=25, em_min_iter=6, em_max_iter=20, chunksize=100):
+        """
+        Sklearn wrapper for LdaSeq model. Class derived from gensim.models.LdaSeqModel
+        """
+        self.corpus = None
```

Comment:
@menshikh-iv Yes, that is true for sklearn. Removing `corpus` attribute from all the wrappers then.